devel/setup-e2e-deps.sh

```diff
@@ -31,25 +31,34 @@ source "${SCRIPT_ROOT}/lib/lib.sh"
 # Configure PATH to use bazel provided e2e tools
 setup_tools
 
-echo "Installing cert-manager into the kind cluster..."
+echo "Installing cert-manager into the cluster..."
 "${SCRIPT_ROOT}/addon/certmanager/install.sh"
 
 check_bazel
 bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //devel/addon/...
 
-echo "Installing sample-webhook into the kind cluster..."
+echo "Installing sample-webhook into the cluster..."
 "${SCRIPT_ROOT}/addon/samplewebhook/install.sh" &
 
-echo "Installing bind into the kind cluster..."
-"${SCRIPT_ROOT}/addon/bind/install.sh" &
+echo "Installing bind into the cluster..."
+if [[ "${IS_OPENSHIFT:-}" == "true" ]] ; then
+  # OpenShift needs bind to be in kube-system due to file ownership restrictions
+  NAMESPACE="kube-system" "${SCRIPT_ROOT}/addon/bind/install.sh" &
```

Comment:
Can we push this switching logic into the `bind/install.sh` script? Otherwise it makes it more complex for users who are 'piece meal' creating development environments (i.e. by calling `create-openshift.sh` followed by `addon/bind/install.sh` etc)